src/AdminConsole/Pages/Organization/Settings.cshtml.cs

```diff
@@ -38,12 +35,27 @@ public class SettingsModel : PageModel
 
     public async Task OnGet()
     {
-        Organization = await _dataService.GetOrganizationWithData();
+        await LoadData();
+    }
+
+    private async Task LoadData()
+    {
+        var organization = await _dataService.GetOrganizationWithData();
+        Name = organization.Name;
+        ApplicationsCount = organization.Applications.Count;
```

Comment:
Why not just set `this.Organization = organization` instead of splitting into separate props? To reduce the amount of data transferred?

Reply:
@Tyrrrz It wouldn't be possible to use Organization as a property (only because we refactored a bit in the last commit!) -  and I think in this case we might have been able to. Otherwise I think it's a good idea to try to be explicit (where it's practical) with what objects we expose to the view and/or modelbinding to add friction to someone accidently adding binding on a sensitive object.